the-simple-workflow.md

```diff
@@ -0,0 +1,61 @@
+# The S.I.M.P.L.E Workflow
+
+The S.I.M.P.L.E workfow provides a guide for how we should be working with clients at Leaf. It’s not design or development specific, but instead focuses on how we work through identifying problems, solutions to those problems, and then delivering those solutions.
+
+The workflow has a few key themes: a **linkable point of reference**, **client communication**, and **seeking help**.
+
+To help it stick, we’ve ~~forced~~ developed a clever acronym for the workflow - S.I.M.P.L.E!
+
+## Specify the problem
+This might be given to you clearly by the client, but they might need you to **communicate with them** to help get a shared clear definition of the problem.
+
+When you feel you have that shared understanding, ensure it’s clearly defined somewhere linkable. This might be a Basecamp message or todo, or a Trello card, or a Pivotal ticket. The important thing is to provide a **linkable point of reference** where someone can check the status of the work.
+
+
+## Identify solutions
+There might be one clear solution to the problem, or several with different benefits and drawbacks. For example:
+
+- If the problem is time sensitive, is there a solution that gets 80% of the problem solved in 20% of the time?
+- If there’s a good solution that doesn’t need to be done all at once, could it be split up into phases?
+- Is there an opportunity to kill two birds with one stone?
+- Can a bit more work now save a lot of pain in the long term?
+
+Unless the solution is completely obvious, **communicate with the client**, combining your expertise with their business knowledge at this stage in order to understand the possible solutions available. You may also need to reach out to your co-workers for assistance.
+
+## Make a joint decision
+Agree with the client on a solution. You may want to **communicate** a few options and have them choose based on benefits and drawbacks, or you might make a recommendation. Ensure the chosen solution is clearly communicated under your **linkable point of reference**.
+
+If the solution isn’t a obvious choice, include the reasoning the chosen solution in your **linkable point of reference**. Reasoning is often forgotten, and is frequently invaluable to refer back to.
```

Comment:
**Suggestion:** highlight key ~~reasoning~~ considerations ~~the chosen solution~~ in ~~your~~ the